patterns/2-structured/project-setup/base-documentation.md

```diff
@@ -81,23 +81,44 @@ topics:
 
 ![CONTRIBUTING.md](../../../assets/img/standard-base-documentation/CONTRIBUTING-for-contributors.png)
 
+### COMMUNICATION.md
+
+Create a separate `COMMUNICATION.md` document. Link this document to your `README.md` so comprehensive contact information can be provided and not take up the extra space in your README. This document should answer frequently
+asked questions about communicating with your team that contributors need to know. The goal is to streamline communications so users and contributors reach out to the correct person through a single channel. This reduces unnecessary distractions for team members and organizes communications so they do not get lost.
+
+Sections in the `COMMUNICATION.md` points of contact for incoming communications and information about outgoing communications from the project's ownership team. Included below are some examples.
```

Comment:
@kschueths I still don't get this sentence:
> Sections in the `COMMUNICATION.md` points of contact for incoming communications and information about outgoing communications from the project's ownership team.

Is there a verb missing?

Reply:
I can't seem to get to the place to edit it. I added a verb "include"

Sections in the `COMMUNICATION.md` include points of contact for incoming communications and information about outgoing communications from the project's ownership team. Included below are some examples.

Reply:
I will add the fix for you.

For next time:
Hit this button here in the comment thread. That will give you a suggestion field within the comment window where you can make your edits. Those can then be committed directly in the comment thread as well.

![Screenshot 2023-09-14 at 21 25 24](https://github.com/InnerSourceCommons/InnerSourcePatterns/assets/163029/553b72e2-2984-46bf-84a6-8a18cd9f1f28)

Reply:
```suggestion
Sections in the `COMMUNICATION.md` include points of contact for incoming communications and information about outgoing communications from the project's ownership team. See some examples below.
```